### ISSUE TITLE
Add a 'Jobs' link to the header to increase click-through

### ISSUE THREAD

**Issue by gregdorward** [redbadger/website-honestly#565]

**Comment by gregdorward** [comment]
These are the results of a comparison between the 2 weeks before, and 2 weeks after the release

- Traffic to the about-us/join-us page has decreased by 0.16%
- Traffic to the other top 5 most popular pages on the site dropped by 30.97% so, in comparison, the join-us page hasn't performed too badly

Navigation to the jobs page* by clicks:

- Jobs from header - 169 (59.5%)
- Jobs form footer - 101 (35.56%)
- Jobs from About-us link - 14 (4.93%)

*Excluded from this list is any navigation to the join-us page through other means, such as through a blog post, or 3rd party site.

In summary, it looks like we've made it easier to get to the job page, with most people using the link in the header to navigate to it, rather than scrolling to the footer, or going through the About us page. There has been no spike in traffic, with a very slight decrease recorded. Given the context of the drop off in traffic across the site, this small decrease could be considered negligible in comparison.

**Comment by gregdorward** [comment]
Since this was released (04/08 - 15/09), we have had 815 visits to the jobs page (through internal, tracked links)

The header link was clicked 504 times 

The footer link was clicked 288 times

The link on the About Us page was clicked 23 times

That equates to:

Header - 61.8% of clicks
Footer - 35.3% of clicks
About us - 2.8% of clicks

22/06 - 03/08 - 3148 page views
04/08 - 15/09 - 3305 page views

An increase in overall page views of about-us/join-us of 4.99%
